apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/ClaimTypeV2.java

```diff
@@ -101,6 +104,42 @@ public enum ClaimTypeV2 {
   /** The entity lazy attributes. */
   private final Collection<PluralAttribute<?, ?, ?>> entityLazyAttributes;
 
+  /**
+   * Returns the {@link ClaimTypeV2} that corresponds to the specified {@link Entity} {@link Class}.
+   *
+   * @param claimTypes {@link Set} set of claim identifiers requested by client
+   * @param val {@link Integer} bitmask denoting the claim types that have data
+   * @return {@link EnumSet} of claim types to process
+   */
+  public static EnumSet fetchClaimsAvailability(Set<ClaimTypeV2> claimTypes, Integer val) {
```

Comment:
So I'd advocate that we leave it here; the MAIN reason, is that we no longer need / support the bifurcation of ClaimType. We can simply have one ClaimType and get rid of the whole v1/v2 in this enum class. Happy to move into a Utils method, but feels like that doesn't provide much if we do cleanup the ClaimType to a single class.

Reply:
I think we could do both. The responsibility of determining the type of claims that exist in a bitmask doesnt seem like it belongs of this enum. However we could still reduce the annoyance of having two enums for maintainability even if this didn't live here. When we eventually condense some of the shared code for v1/v2, a shared utils class makes sense to me, where this would live nicely.